packages/cxl-institute/scss/global.scss

```diff
@@ -96,20 +96,20 @@ body {
   object,
   embed {
     display: flex;
-    height: 500px;
+    height: 250px;
     margin: 0 auto;
     width: 100%;
 
-    @include mq($until: wide) {
-      height: 400px;
+    @include mq($from: wide) {
+      height: 500px;
```

Comment:
What sense does "going in reverse" make?

"Mobile first" should mean "smaller to bigger" across the specturm.